doc/compression.md

```diff
@@ -0,0 +1,13 @@
+# Adjusting compression
+
+The environment variable `instsys_no_compression` accepts a comma-separated
+list of values to omit compression in certain places.
```

Comment:
That's not how it's actually parsed but whatever. Passing what is documented should work.

Reply:
As above.